models/src/main/java/com/vimeo/networking2/AlbumList.kt

```diff
@@ -0,0 +1,33 @@
+package com.vimeo.networking2
+
+/**
+ * List of albums that could be paged.
+ */
+data class AlbumList(
+
+    /**
+     * Total number of albums.
+     */
+    override val total: Int? = null,
+
+    /**
+     * The current page number of paging list.
+     */
+    override val page: Int? = null,
+
+    /**
+     * The number of albums to returns per page.
+     */
+    override val perPage: Int? = null,
+
+    /**
+     * Urls to the first, last page, next and previous pages.
+     */
+    override val paging: Paging? = null,
+
+    /**
+     * List of albums.
+     */
+    override val data: List<Album>?
```

Comment:
There's a lot of `Pageable` implementations that have a default value of null here and a lot that don't. We should be consistent about whether there are default values for these properties.